recipes/obspy/run_test.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib
+matplotlib.use("AGG")
+from obspy.core import run_tests
+errors = run_tests(report=True, hostname="conda-forge", verbosity=2)
```

Comment:
@ocefpaf, yes, we do run CI (Appveyor, Travis).

In general I'm a bit paranoid when distributing packages and like to run the tests directly from the built packages because testing the packages (from release tarballs) is somewhat different than testing the github repo (at one point we had to do one release over and over again several times because of problems with missing files).

However, if you feel our build is eating too much of your CI I can of course deactivate the full test suite.